### PR TITLE
Add (Currently broken.) after "Enable multi-core"

### PR DESCRIPTION
**Pull request by MysticExile**
Should be good now

### CODE REVIEW

File: src/yuzu/configuration/configure_general.ui [hunk 1]
```diff
@@ -61,7 +61,7 @@
           <item>
            <widget class="QCheckBox" name="use_multi_core">
             <property name="text">
-             <string>Enable multi-core</string>
+             <string>Enable multi-core (Currently broken.)</string>
```
Comment (bunnei) [inline src/yuzu/configuration/configure_general.ui]: Should be:
`Enable multi-core (currently broken)`

**Comment by bunnei** [comment]
On second thought, let's just remove this from the UI for now. (We should still leave the config setting, FWIW... it just shouldn't be exposed to users at all if its broken). Thanks!

**Comment by MysticExile** [comment]
@bunnei I’ll close it for now then. And I’ll open a new PR later this day to entirely remove the option from the UI. Thanks for reviewing!

**Comment by jroweboy** [comment]
@bunnei removing the option from the UI means that for the small number of people that have launched citra in the past with the option enabled, they won't be able to turn it off without wiping their config

**Pull request closed without merging**